gpu_direct_rdma_access.c

```diff
@@ -838,6 +866,66 @@ struct rdma_device *rdma_open_device_server(struct sockaddr *addr)
     return NULL;
 }
 
+int rdma_reset_server_device(struct rdma_task_attr *attr)
+{
+	struct rdma_device *device = attr->local_buf_rdma->rdma_dev;
+	if (is_server(device)) {
```

Comment:
the correct check should be "if (!is_server()))..."
or you can rename it to be is_client()